CoreWidgetProvider/Widgets/SSHWalletWidget.cs

```diff
@@ -36,6 +37,8 @@ public SSHWalletWidget()
 
     ~SSHWalletWidget()
     {
+        // Ensures widget is in the proper view when a user returns
+        UpdateWidget();
```

Comment:
Is the destructor only called on the 'remove widget' event? I was trying to ensure that if a user gets into a pattern matching state but didn't end up connecting to an SSH connection that if they were to close Dev Home and return the json that is being shown is the original one with the list of ssh connections rather than the pattern matching template. If the destructor is only called on the 'Remove Widget' action then I think this is irrelevant and I will just remove it.

Reply:
I expect that the next time the constructor is called for this widget, it will be created in the right state, so this shouldn't be needed here. If that's not the case, probably a thing to fix :)